### DISCUSSION TITLE
rockmqt 5.1.4正常运行了很长时间，最近出现了某个队列无法消费的情况，大家有遇到过吗？

### DISCUSSION THREAD

**Discussion by maxid** [apache/rocketmq#9000] [General]
![WX20241128-230127](https://github.com/user-attachments/assets/50c1bc5b-b3e7-4f0c-b316-25a8c7498f7c)
如上图所示，可以确认消费端没有问题，就是无法消费

**Comment by pxd98** [comment]
这个broker是不是有问题，或者重启过？我之前遇到过一个问题，就是broker重启后发现这个broker不会写入新消息，是因为里面的消息都过期很久了，store这个目录里有过期数据，把这个目录删了后重建就可以了。

**Comment by humkum** [comment]
这种一般是消费线程卡住了，可以截一下消费客户端的 jstack 看下
